Dshop/apps/products_catalogue/urls.py

```diff
@@ -2,13 +2,12 @@
 from django.conf.urls.static import static
 from django.urls import path
 
-from .views import CeneoProductListView, ProductListView, \
+from .views import CeneoProductListView, \
     ProductDetailView, AddToCartView, CartDetailView, CategoryListView, CategoryDetailView, DeleteOneCartItemView, \
     DeleteCartItemView
 
 
 urlpatterns = [
-    path('products_list/', ProductListView.as_view(), name='products-list'),
     path('ceneo/', CeneoProductListView.as_view(), name='product-list'),
```

Comment:
```suggestion
    path('products/ceneo/', CeneoProductListView.as_view(), name='product-list'),
```
Since all links except main page are prefixed with "products", let's also add it here.